### PR TITLE
Adding list of Schools in training page

### PR DESCRIPTION
**Pull request by GuillermoFidalgo**

### CODE REVIEW

**Comment by GuillermoFidalgo** [comment]
Let me know what needs to be fixed.

File: Schools/_posts/2018-09-25-AEPSHEP.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (graeme-a-stewart) [inline Schools/_posts/2018-09-25-AEPSHEP.md]: So these `_posts` entries do generate webpages that can be accessed from the navigation through all of the posts that HSF makes (meetings, announcements, etc). 

e.g., this one is accessible at <http://127.0.0.1:4000/schools/2018/09/25/AEPSHEP.html> in a local Jekyll environment.

So you need to make sure they have a link to the original school (and it's better to just have a link than be repetitive of information that's elsewhere).

File: Schools/_posts/2019-02-01-CMS-SHERPA.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: The First CMS SHERPA Workshop
+source: https://indico.cern.ch/event/779232/
+date: 2019-02-01
+author: Guillermo Fidalgo
+layout: event
+categories: Schools
+---
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```
Comment (graeme-a-stewart) [inline Schools/_posts/2019-02-01-CMS-SHERPA.md]: Ditto here - should be a link text to the school.

File: Schools/_posts/2019-02-02-Trans-Siberian-School.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: 2nd Trans-Siberian School on High Energy Physics
+source: http://hep2019.tpu.ru/webcenter/portal/transibeng/home?_adf.ctrl-state=vcg8rk92x_4
+date: 2019-04-01
+author: Guillermo Fidalgo
+---
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```
Comment (graeme-a-stewart) [inline Schools/_posts/2019-02-02-Trans-Siberian-School.md]: Ditto

File: Schools/_posts/2019-02-03-Summer-CTEQ-School.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+title: 2019 CTEQ Summer School University of Pittsburgh, USA
+source: http://www.physics.smu.edu/scalise/cteq/
+date: 2019-07-16
+author: Guillermo Fidalgo
+---
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```
Comment (graeme-a-stewart) [inline Schools/_posts/2019-02-03-Summer-CTEQ-School.md]: Ditto

File: Schools/_posts/2019-03-13-Latin-American-School.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+title: Latin American School
+source: https://indico.cern.ch/event/763013/
+date: 2019-03-13
+author: Guillermo Fidalgo
+---
+<!-- # [Click here for more information](https://indico.cern.ch/event/763013/)
+
+ ![]({{site.baseurl}}/assets/img/posts/Latin_school_Banner.png) -->
+
+Welcome to the 10th CERN Latin-American School of High-Energy Physics (CLASHEP2019).
```
Comment (graeme-a-stewart) [inline Schools/_posts/2019-03-13-Latin-American-School.md]: Ditto

File: Schools/_posts/2019-06-02-INFN_School.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
+title: INFN School of Statistics
+source: https://agenda.infn.it/event/16360/
+date: 2019-06-02
+author: Guillermo Fidalgo
+---
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```
Comment (graeme-a-stewart) [inline Schools/_posts/2019-06-02-INFN_School.md]: Ditto

File: Schools/_posts/2019-08-28-14th-CERN-Fermilab-HCPSS.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+---
+title: 14th CERN-Fermilab Hadron Collider Physics Summer School
+source: https://indico.cern.ch/event/795313/
+date: 2019-08-28
+author: Guillermo Fidalgo
+---
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```
Comment (graeme-a-stewart) [inline Schools/_posts/2019-08-28-14th-CERN-Fermilab-HCPSS.md]: Ditto

File: Schools/_posts/2019-09-01-European-School.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+---
+title: European School
+source: http://physicschool.web.cern.ch/PhysicSchool/ESHEP/default.html
+date: 2019-09-01
+author: Guillermo Fidalgo
+---
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```
Comment (graeme-a-stewart) [inline Schools/_posts/2019-09-01-European-School.md]: Ditto

**Comment by graeme-a-stewart** [comment]
You could also add a new page, just for listing training schools and events (say `Schools/events.md`) that contains these posts (in the same way that you did on the training page).

Then we can link to that page from the "Communication" drop down menu (see `_includes/navbar.ext` for how to do that).

File: Schools/_posts/2018-09-25-AEPSHEP.md (hunk 1 above)
Comment (GuillermoFidalgo) [reply Schools/_posts/2018-09-25-AEPSHEP.md]: Is this what you meant? 

```suggestion
---
title: AEPSHEP (Asia-Europe-Pacific School of High-Energy Physics)
source: http://2018.aepshep.org/
date: 2018-09-25
author: Guillermo Fidalgo
---
[AEPSHEP](http://2018.aepshep.org/)

```

**Comment by GuillermoFidalgo** [comment]
I think I have made the necessary fixes. Let me know if there is some detail I've missed.

**Comment by GuillermoFidalgo** [comment]
Hey @graeme-a-stewart  I think I have made the necessary changes but I dont quite understand why it says "All checks have failed". I have a lot of content that is ready to be pushed. Let me know if I missed something. Many thanks 
-Guillermo

File: Schools/_posts/2019-02-01-CMS-SHERPA.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
+---
+title: The First CMS SHERPA Workshop
+source: https://indico.cern.ch/event/779232/
```
Comment (graeme-a-stewart) [inline Schools/_posts/2019-02-01-CMS-SHERPA.md]: This is a restricted URL (probably only CMS people can see it). So for the HSF site you'd need to find some public URL that's useful or remove this entry.

Comment (GuillermoFidalgo) [reply Schools/_posts/2019-02-01-CMS-SHERPA.md]: done

File: Schools/_posts/2019-02-02-Trans-Siberian-School.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
+---
+title: 2nd Trans-Siberian School on High Energy Physics
+source: http://hep2019.tpu.ru/webcenter/portal/transib2019/page11?_adf.ctrl-state=3sr6xde5g_57
```
Comment (graeme-a-stewart) [inline Schools/_posts/2019-02-02-Trans-Siberian-School.md]: Just one last thing - that URL looks very odd. Surely the POST information shouldn't be there. i.e., it should just be

<http://hep2019.tpu.ru/webcenter/portal/transib2019/page11>

Comment (graeme-a-stewart) [reply Schools/_posts/2019-02-02-Trans-Siberian-School.md]: OK, it was such a small change I just did it myself...

**Comment by GuillermoFidalgo** [comment]
Thanks Graeme!!

**Comment by graeme-a-stewart** [comment]
OK, last URL failures are all spurious so good to merge

**Pull request merged**